docs/monitors/elasticsearch.md

```diff
@@ -11,77 +11,116 @@ This monitor collects cluster level and index level stats only from the current
 in an Elasticsearch cluster by default. It is possible to override this with the
 `clusterHealthStatsMasterOnly` and `indexStatsMasterOnly` config options respectively.
 
-A simple configuration that collects only default metrics looks like the following
+A simple configuration that collects only default, included metrics looks like the following:
```

Comment:
We've started to use the term "bundled metrics" in the docs, in place of standard metrics or, in this case, default included metrics. So to start letting people know that, this could say something like:

A simple configuration that collects only bundled metrics (non-custom metrics included by default)  looks like the following:

If you were really ambitious, you could even link to the docs: https://docs.signalfx.com/en/latest/admin-guide/usage.html#about-custom-high-res

Reply:
@barbara-sfx @jadeblaquiere I have been working with Joe deB on the monitor template. He prefers the word "default" metrics, as opposed to "optional" metrics. Maybe you two should talk and decide on wording.

Reply:
_Default_ is a good word given how our new filtering stuff will truly make those bundled metrics also the default config.